graphics/macgui/mactext.cpp

```diff
@@ -2089,30 +2089,28 @@ void MacText::getRowCol(int x, int y, int *sx, int *sy, int *row, int *col) {
 
 	nsy = _textLines[nrow].y;
 
-	ncol = 0;
+	if (_textLines[nrow].chunks.size() > 0) {
```

Comment:
This is a redundant check and adds extra indentation. please revert. the loop will not run then

Reply:
It's not redundant, it allows the later check for "chunk" being in-range to be removed, and allows some unnecessary work (like calling getAlignOffset) to be skipped.